openupgradelib/openupgrade.py

```diff
@@ -1056,12 +1060,16 @@ def wrapped_function(cr, version):
                 logger.info(
                     "%s: %s-migration script called with version %s" %
                     (module, stage, version))
+                # Set up useful default context values
+                context = context or {}
+                context['migrate_version'] = version and version or None
+                context['module_name'] = module and module or None
```

Comment:
As a side note: The last time I proposed interdependent changes, it resulted in days wasted in useless refactoring, as anyhow no duly consensus could be drawn and my own fork and upstream code were terribly torn apart and almost impossible to integrate. Yet, completely walking away is not yet warranted and not a real option, anyhow... :smile: